src/Object/Contact.php

```diff
@@ -746,8 +721,7 @@ public static function getPostsFromUrl($contact_url)
 
 		// There are no posts with "uid = 0" with connector networks
 		// This speeds up the query a lot
-		$r = q("SELECT `network`, `id` AS `author-id`, `contact-type` FROM `contact`
-		WHERE `contact`.`nurl` = '%s' AND `contact`.`uid` = 0", dbesc(normalise_link($contact_url)));
+		$r = dba::select('contact', array('network', 'id AS author-id', 'contact-type'), array('contact.nurl' => normalise_link($contact_url), 'contact.uid' => 0));
```

Comment:
I really don't think that this can work.

And yeah I guess I have to write a documentation for the dba methods ;-)

Reply:
There's no need for table prefix since there's only one table selected. So this could work without the prefixes.

Reply:
I thought so too, but it was in the previous code and I left it.

Reply:
Of course, but I don't think the `select()` condition array accepts table prefixes just yet.